app/src/main/java/org/mozilla/fenix/perf/StartupTimeline.kt

```diff
@@ -41,10 +31,7 @@ object StartupTimeline {
     private var state: StartupState = StartupState.Cold(StartupDestination.UNKNOWN)
 
     private val reportFullyDrawn by lazy { StartupReportFullyDrawn() }
-    internal val frameworkStartMeasurement by lazy { StartupFrameworkStartMeasurement() }
-    internal val homeActivityLifecycleObserver by lazy {
-        StartupHomeActivityLifecycleObserver(frameworkStartMeasurement)
-    }
+    internal val frameworkStartMeasurement by lazy { ApplicationInitTimeContainer() }
```

Comment:
nitpick (non-blocking): ideally we'd change this property name too but I don't want to delay us further on another review. Feel free to fix it in a subsequent PR.

Reply:
Yes, I was not sure what to name it. Could add it in `cleaning unused resources PR`

Reply:
Usually a good choice is naming it after the class, e.g.:
```kotlin
internal val applicationInitTimeContainer by lazy { ApplicationInitTimeContainer(...) }
```

> Could add it in cleaning unused resources PR

Sounds good. btw, in that "unused resources" PR, please don't remove any unused strings. We have to figure out how unused strings will work with our localization workflow: https://github.com/mozilla-mobile/fenix/issues/20871